.travis.yml

```diff
@@ -26,6 +26,7 @@ script:
  - ./configure --disable-dependency-tracking --enable-minimal
  - make check
  - ( echo '#include <sodium.h>' ; echo 'int main(void) { return sodium_init(); }' ) > /tmp/main.c && gcc -Isrc/libsodium/include -Isrc/libsodium/include/sodium $(find src -name '*.c' -o -name '*.S') /tmp/main.c
+ - cd fuzz && make fuzz-check
```

Comment:
It might be simpler/more straightforward just to make another row in the build matrix that runs the fuzz-check rule.